### PR TITLE
ARM SiteRecovery cmdlets

### PR DESCRIPTION
**Pull request by sriramvu**
Hi, this PR has ARM SiteRecovery cmdlets.

Updated PR with published NuGet.

Thank you.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@sriramvu__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (sriramvu). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/AzurePowershell.sln [hunk 1]
```diff
@@ -1,7 +1,5 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
```
Comment (ljhljh235) [inline src/AzurePowershell.sln]: This should not be changed.

File: src/ResourceManager/Resources/Commands.Resources/AzureResourceManager.psd1 [hunk 2]
```diff
@@ -103,7 +103,8 @@ NestedModules = @(
     '.\OperationalInsights\Microsoft.Azure.Commands.OperationalInsights.dll',  
     '.\UsageAggregates\Microsoft.Azure.Commands.UsageAggregates.dll',  
     '.\ApiManagement\Microsoft.Azure.Commands.ApiManagement.ServiceManagement.dll',  
-    '.\AzureBackup\Microsoft.Azure.Commands.AzureBackup.dll'  
+    '.\AzureBackup\Microsoft.Azure.Commands.AzureBackup.dll',
+	'.\SiteRecovery\Microsoft.Azure.Commands.SiteRecovery.dll'
```
Comment (ljhljh235) [inline src/ResourceManager/Resources/Commands.Resources/AzureResourceManager.psd1]: Mixed tabs and spaces.

File: src/ResourceManager/SiteRecovery/Commands.SiteRecovery.Test/ScenarioTests/vaultSettings.VaultCredentials [hunk 3]
```diff
@@ -0,0 +1 @@
...i7c4DlF9cg5vXei/4NQ2rIJby2nquwzjFAl43hlPRb8zQQ80s0HLMcnDWkEFpKIpqlElTDjj6InUgI+eY1N+SHrzzXvV9+MdOZ2Z/0Pbveqt+7JiHCl2lwWDIaw5210S7LQnqCwkqVwS0o3wnGE18I4Y6mCnu8nFwafEinWETUXwc/iMHKl3GPkocSvkULG+ZlS+Cn++jvPimmc2I2PhnSli2ChU47/3yOH0zjRurGPuxdaZd2UcKcCvZOVNnTYMDJAHICgigElTBvZnsZIOG3rByg/06kYaPn5Acd6nT2KfMj2lkc9oAeFn6qXx2GUkq9BV3VrFEm/ugmqZ7iHT/8PVdJXLoUSx/Q64CENAN2cg+jJyC/Mvbe2ihO1mjtPkqtVm05ryUvf+0PcFYuj/eXCX7rKSLiai1tgwVIGUWK0wd5332OxWWRM1fRktAnSfWcF3Mrb100IQBW0X6MSBK2suJYWDjk5S0lQOozEFUugIk9jMwZEAvrmdvOyi4cOgo5MBx34hA0nRebe0LZTlKKXlO1kIURDRIlUDvkEQ8Ny8u0DkywN9bdF4VknmtZDNTzIjBavkYqQGZkYweA5mgfOSlS1FO1KQl7JqtOiisDf6IIvZahwSCGHk9C5eXYsFNkGcHlkGdEUhGwy2hwWkpO1JF36Pw/QCa7mfauXfCFTR/wB69Xc8gfF3g4Mt9BWNVIyj8tiWB23lNt5j7GeB+AV2DKuNqA6Ef7IUd5J+XreKud7ZT9DaM5oOSMDcwHzAHBgUrDgMCGgQUWDFNbl+67TAhAUdLuFwilA75T/4EFMDc7YbA4AZ8CyQ5tIdLT5NdSopi</ManagementCert><AcsNamespace><HostName>accesscontrol.windows.net</HostName><Namespace>wusppe1rrp1users</Namespace><ResourceProviderRealm>http://windowscloudbackup/m3</ResourceProviderRealm></AcsNamespace><ChannelIntegrityKey>tmPfTki5UFSdaEq2JFvzuw==</ChannelIntegrityKey><ResourceGroupName>testsitegroup</ResourceGroupName><Version>1.0</Version></ASRVaultCreds>
```
Comment (ljhljh235) [inline src/ResourceManager/SiteRecovery/Commands.SiteRecovery.Test/ScenarioTests/vaultSettings.VaultCredentials]: Is the cert string here by design?

Comment (sriramvu) [inline src/ResourceManager/SiteRecovery/Commands.SiteRecovery.Test/ScenarioTests/vaultSettings.VaultCredentials]: we need that to set the vault context.

**Comment by ljhljh235** [comment]
I don't see any tests being run here: http://azuresdkci.cloudapp.net/job/powershell/1940/artifact/src/Publish/TestResults/Microsoft.Azure.Commands.RecoveryServices.Test.html

Could you verify the tests to run on the CI server?

**Pull request merged**